backend/post/pagination.py

```diff
@@ -0,0 +1,7 @@
+from rest_framework import pagination
+
+class CustomPageNumberPagination(pagination.PageNumberPagination):
+    page_size = 1
+    page_size_query_param = 'count'
+    max_page_size = 5
+    page_query_param = 'p'
```

Comment:
```suggestion
    page_query_param = 'page'
```
I think this should be page, can you test and confirm.
Also I noticed that your next url looks like this `...&p=2@page=1`, maybe this is the reason

Reply:
I customized it po so that either "page" or "p" could work in both cases. But I think your suggestion is better po and more readable. Thank you sir.

Reply:
Just use 1 parameter to avoid confusion

Reply:
Noted po. Thank you sir.